test/runtests.jl

```diff
@@ -13,7 +13,13 @@ tests = ["2by2.jl",
 
 println("Running tests:")
 
-for t in tests
-    println(" * $(t)")
-    include(t)
+for test in tests
+    try
```

Comment:
Why do we now need this try/catch?

Are some tests now failing that were previously passing?